python/GafferSceneUI/PathFilterUI.py

```diff
@@ -114,6 +118,56 @@
 
 )
 
+##########################################################################
+# VectorDataPlugValueWidget customisation
+###########################################################################
+
+class _PathsPlugValueWidget( GafferUI.VectorDataPlugValueWidget ) :
+
+	def __init__( self, plug, **kw ) :
+
+		GafferUI.VectorDataPlugValueWidget.__init__( self, plug, **kw )
+
+		self.vectorDataWidget().dataMenuSignal().connect( Gaffer.WeakMethod( self.__dataMenu ), scoped = False )
+
+	def __dataMenu( self, vectorDataWidget, menuDefinition ) :
+
+		selectedIndices = vectorDataWidget.selectedIndices()
+
+		if len( selectedIndices ) > 0 :
+			menuDefinition.append( "/divider", { "divider" : True } )
+			menuDefinition.append(
+				"/Select Filtered Objects",
```

Comment:
I know I dithered around on what this should be named, but I now think it should be called "Select Affected Objects", to match the equivalent menus elsewhere. I'm not sure it's the best name, but my being consistent with existing functionality I think folks will get it more easily...

Reply:
Fixed in 533323a0eca795acb5fd1166f3b2dea6fd529a53